webapp/app/components/Login/index.js

```diff
@@ -3,64 +3,28 @@
 //
 
 import React from 'react';
-import { FormGroup, FormControl, ControlLabel, Image } from 'react-bootstrap';
-import { LinkContainerButton } from '../Button';
-import { browserHistory } from 'react-router';
+import { PageHeader, Image } from 'react-bootstrap';
+import LoginForm from 'containers/Login/Form';
 import Logo from '../App/logo_sam_solution.png';
```

Comment:
Put 'components/App/logo_sam_solution.png' instead of '../App/logo_sam_solution.png'